packages/adapters/ethers/src/client.ts

```diff
@@ -465,7 +465,7 @@ export class EthersAdapter extends AdapterBlueprint {
     params: AdapterBlueprint.GetBalanceParams
   ): Promise<AdapterBlueprint.GetBalanceResult> {
     const address = params.address
-    const caipNetwork = this.caipNetworks?.find(network => network.id === params.chainId)
+    const caipNetwork = this.getCaipNetworks().find(network => network.id === params.chainId)
```

Comment:
should we remoev the `caipNetworks` property alltogether? else it would be confusing to know which to use

Reply:
Exactly